src/Vizkit3dWorld.cpp

```diff
@@ -155,6 +164,14 @@ vizkit3d::RobotVisualization* Vizkit3dWorld::robotVizFromSdfModel(sdf::ElementPt
     //vizkit3d plugin with model defined in the sdf model
     vizkit3d::RobotVisualization* robotViz = new vizkit3d::RobotVisualization();
 
+    for (int i=0; i < excludeFromSdf.size(); ++i){
```

Comment:
It's weird that the camera frame changes to flat_fish. There's no reason for that since we definitely do NOT place the camera there. Would be good to know why this error pops up.